frontend/docs/examples.md

```diff
@@ -0,0 +1,31 @@
+---
+id: examples
+title: Examples
+hide_title: false
+hide_table_of_contents: false
+keywords:
+  - final space
+  - finalspace
+  - finalspaceapi
+  - final space wiki
+  - api
+  - restapi
+  - rest api
+  - examples
+description: Examples section for Final Space.
+---
+
+### Examples Section
+
+Below is a video from youtuber Weibenfalk on how to use data from the Final Space API to populate a React Slider Component that he built a while ago.
+
+[![React Slider with Final Space API](https://img.youtube.com/vi/zmwG5lYQYRo/0.jpg)](https://www.youtube.com/watch?v=zmwG5lYQYRo "React Slider with Final Space API")
```

Comment:
This is an image with link

Have you tried embedding the video itself?

The issue was for embedding video,

Reply:
We can't directly embedded youtube video links.